static/custom_dc/unsdg/css/custom.css

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.header-container {
```

Comment:
Maybe add a comment to sync this to the unsdg css file

Reply:
Which file are you referring to here?

Reply:
I thought you have a PR to take the css out from Dan's sdg tracker app.

Reply:
Ah - yes, this CSS is copied from the tracker [code][code]. Ideally, we'd want both to be the same code rather than keeping them in sync if possible. I'll "sync" with @dwnoble on this :)

[code]: https://github.com/datacommonsorg/website/tree/master/experimental/sdg-tracker/src/components/layout